docs/onboarding_to_argocd.md

```diff
@@ -0,0 +1,123 @@
+# ArgoCD onboarding procedure
+
+We have an ArgoCD instance deployed on MOC that can be used to deploy the application manifests located in this repo.
+
+The ArgoCD instance is deployed on [MOC CNV Sandbox](https://github.com/open-infrastructure-labs/moc-cnv-sandbox) cluster.
+
+Each team is given an [ArgoCD Project](https://argoproj.github.io/argo-cd/user-guide/projects/) that has an allow-list of clusters and namespaces to which they can deploy. They also have an allow-list of resources they can deploy within those namespaces. These restrictions exist to prevent teams from being able to use ArgoCD to deploy cluster scoped resources, or resources onto other team's namespaces.
+
+> Currently, you can only use this ArgoCD instance to deploy to moc-cnv-sandbox cluster, other clusters are not yet supported, please make an issue if you have this requirement.
+
+Team's are given edit access (via ui/cli console) to their ArgoCD Projects using OpenShift RBAC and OpenShift Groups.
+
+The following steps should be completed to fully onboard and enable a team to use ArgoCD to deploy `Applications` to their ArgoCD `Projects`.
+
+> Note: ArgoCD Projects, should not to be confused with OpenShift Projects.
+
+## Pre-requisites
+Team requesting ArgoCD access must have been onboarded to the cluster. See [here](https://github.com/operate-first/support/blob/main/docs/onboarding_to_cluster.md)
+
```

Comment:
I'd add to fork/checkout the `argocd-apps` and `continuous-delivery` repos, since this doc is making changes to those particular repos and not the `support`.

Reply:
updated